emulate-browsers/safari-13/lib/helpers/configureSessionTls.ts

```diff
@@ -0,0 +1,5 @@
+import { BrowserEmulatorBase } from "@secret-agent/browser-emulator-utils";
+
+export default function configureSessionTcp(emulator: BrowserEmulatorBase, settings) {
+  settings.tlsClientHelloId = emulator.userAgentOption.browserId.match(/^([a-z]+-[0-9]+)/)[1];
```

Comment:
Just another place that makes me wonder if this is actually working in Go tls sockets.